reporter/html.js

```diff
@@ -63,6 +63,7 @@ if ( typeof window === "undefined" ) {
 }
 
 var config = QUnit.config,
+	collapse = config.collapse,
```

Comment:
This makes it impossible to actually change the behaviour via `QUnit.config`, since any changes to `QUnit.config.collapse` after `qunit.js` was loaded will be ignored. Tested by adding this to the top of `test/test.js`:

```js
QUnit.config.collapse = false;
```

Should be easy to fix by not making a copy of `config.collapse`, but still using a local variable to keep track of the need to collapse or not.